aries/aries_vcx_core/src/wallet2/indy_wallet/indy_did_wallet.rs

```diff
@@ -0,0 +1,145 @@
+use async_trait::async_trait;
+use vdrtools::{DidMethod, DidValue, KeyInfo, Locator, MyDidInfo};
+
+use crate::{
+    errors::error::{AriesVcxCoreError, VcxCoreResult},
+    wallet2::{DidData, DidWallet},
+};
+
+#[async_trait]
+impl DidWallet for crate::wallet::indy::IndySdkWallet {
```

Comment:
To stay consistent with most of the codebase, import `IndySdkWallet` rather than fully qualify directly in the code